Tools/scripts/sitl-on-hardware/sitl-on-hw.py

```diff
@@ -82,12 +152,14 @@ def sohw_path(fname):
 configure_args.extend(unknown_args)
 run_program(configure_args)
 
-build_cmd = ["./waf", args.vehicle]
+# Remove "Ardu", "Heli", and "Antenna"
+waf_vehicle = args.vehicle.lower().replace("ardu", "").replace("heli", "").replace("antenna", "")
```

Comment:
Uh..... how can we remove "Heli" here?  If someone says `./waf helicopter`, they don't want `waf copter`, surely?

Reply:
.... also, does `./waf tracker` actually work?  Doesn't that also need to be `./waf antennatracker`?

Reply:
yep, right, I tottally overlook this . I will correct that

Reply:
done